src/resources/js/views/month-mobile-events.js

```diff
@@ -140,6 +188,8 @@ tribe.events.views.monthMobileEvents = {};
 			$content = $container.find( '#' + contentId );
 		}
 
+		obj.handleMobileDayClick( contentId );
```

Comment:
`handleMobileDayClick()` expects a `boolean` as the first argument, but `contentId` is a string, is this what you intended?